app/code/core/Mage/Wishlist/Model/Wishlist.php

```diff
@@ -74,31 +74,124 @@ protected function _construct()
         $this->_init('wishlist/wishlist');
     }
 
+    /**
+     * Set date of last update for wishlist
+     */
+    #[\Override]
+    protected function _beforeSave()
+    {
+        $this->setUpdatedAt(Mage::getSingleton('core/date')->gmtDate());
+        return parent::_beforeSave();
+    }
+
+    /**
+     * Save related items
+     */
+    #[\Override]
+    protected function _afterSave()
```

Comment:
can we add return type here?

Reply:
These are just moved methods to more closely match `Mage_Sales_Model_Quote`. 

However for research we have `class Mage_Wishlist_Model_Wishlist extends Mage_Core_Model_Abstract` and:

```php
abstract class Mage_Core_Model_Abstract extends Varien_Object
{
    /**
     * Processing object after save data
     *
     * @return $this
     */
    protected function _afterSave()
    {
        Mage::dispatchEvent('model_save_after', ['object' => $this]);
        Mage::dispatchEvent($this->_eventPrefix . '_save_after', $this->_getEventData());
        return $this;
    }
```

I think we could still technically add the return type, but so much code probably extends `Mage_Wishlist_Model_Wishlist` that I'm not sure how beneficial it is.